src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -28,7 +28,6 @@
       (let [{:keys [name color formatted-balance
                     watch-only?]} (rf/sub [:wallet/current-viewing-account])
             customization-color   (rf/sub [:profile/customization-color])]
-        (rn/use-unmount #(rf/dispatch [:wallet/close-account-page]))
```

Comment:
Was this redundant?

Reply:
Hey @smohamedjavid 
I removed it because of:
Please check: https://github.com/status-im/status-mobile/pull/20183#discussion_r1619394309

and the call to this event is already handled in the `on-press` callback for the back button.